src/v/pandaproxy/probe.cc

```diff
@@ -55,4 +56,24 @@ probe::probe(
       sm::impl::default_handle() + 1);
 }
 
+error_probe::error_probe(const ss::sstring& group_name)
+  : _error_count(0)
+  , _metrics() {
+    if (config::shard_local_cfg().disable_metrics()) {
+        return;
+    }
+
+    namespace sm = ss::metrics;
+
+    _metrics.add_group(
+      group_name,
+      {sm::make_counter(
+        "request_errors_total",
+        [this] { return _error_count; },
+        sm::description(ssx::sformat("Total number of {} request errors", group_name)),
+        {},
+        {sm::shard_label})},
+      sm::impl::default_handle() + 1);
```

Comment:
I think this handle got a name in `ssx/metrics.h`

Reply:
done